sdk/core/azure-core/src/main/java/com/azure/core/models/CloudEventDataFormat.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models;
+
+/**
+ * The data part of a {@link CloudEvent} can be serialized to base64 bytes, JSON, or String when a CloudEvent
+ * is serialized.
+ * Use {@link #BYTES} if you'd like to serialize the CloudEvent data to base64 bytes. Use {@link #JSON} otherwise.
```

Comment:
I would change this up a little to discuss more the purpose of this enum to start

```suggestion
 * Representation of the data format for a {@link CloudEvent}.
 * <p>
 * When constructing a {@link CloudEvent} this is passed to determine the serialized format of the event's data. If {@link #BYTES} is used the data will
 * be stored as a Base64 encoded string, otherwise it will be stored as a JSON serialized object.
```

Reply:
Thanks. Updated.